prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -92,6 +92,30 @@ postsubmits:
       - name: docker-root
         emptyDir: {}
 
+  - name: post-test-infra-upload-testgrid-config
+    cluster: test-infra-trusted
+    run_if_changed: '^(prow/cluster/jobs/.*\.yaml)|(testgrid/default\.yaml)$'
+    decorate: true
+    branches:
+    - master
+    annotations:
+      testgrid-create-test-group: "false"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+        command:
+        - ./testgrid/transfigure.sh
+        args:
+        - /etc/github-token
```

Comment:
I suspect this should be `/etc/github-token/oauth`

Reply:
Fixed